lib/factory_bot_rails/reloader.rb

```diff
@@ -31,11 +32,11 @@ def reloader_class
     end
 
     def register_reloader(reloader)
-      config.to_prepare do
+      app.reloaders << reloader
+
+      app.reloader.to_prepare do
```

Comment:
The way I understand this, the ```to_prepare``` block that was part of the config before now will be under the reloaders?

Reply:
In later versions of Rails this amounts to the same thing. `config.to_prepare` [pushes the callbacks into an array](https://github.com/rails/rails/blob/master/railties/lib/rails/railtie/configuration.rb#L79-L83) which later gets [copied over to `app.reloader.to_prepare`](https://github.com/rails/rails/blob/master/railties/lib/rails/application/finisher.rb#L104-L108).

But it didn't work quite like this in Rails 4.2, which is why we are getting CI failures.